src/test/java/org/jabref/logic/importer/fileformat/CffImporterTest.java

```diff
@@ -83,4 +85,122 @@ public void testImportEntries() throws IOException, URISyntaxException {
         assertEquals(entry.getField(StandardField.VERSION), Optional.of("1.0"));
 
     }
+
+    @Test
+    public void testImportEntriesMultipleAuthors() throws IOException, URISyntaxException {
+        Path file = Path.of(CffImporterTest.class.getResource("CffImporterTestValidMultAuthors.cff").toURI());
+        List<BibEntry> bibEntries = importer.importDatabase(file, StandardCharsets.UTF_8).getDatabase().getEntries();
+        BibEntry entry = bibEntries.get(0);
+
+        assertEquals(entry.getField(StandardField.AUTHOR), Optional.of("Joe van Smith and Bob Jones, Jr."));
```

Comment:
Just a litte bit more simpler solution. So you don't need an assertEquals in every line
Create a new BibEntry expected with the fields  and then compare it to the response
BibEntry expected = new BibEntry(...).withField(....)
Collection.singletonList(expected, bibEntries)